dev/staticRuleset.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
should this file even be present in MFTF? Even for standalone version, shouldn't MFTF read the file from magento dev/tests/acceptance path?

Reply:
So we have a global `TESTS_BP` that changes depending on standalone vs embedded, and in embedded it's `dev/tests/acceptance`, that's why I opted to use it. Standalone will have TESTS_BP equal to `dev/tests/acceptance` if there's a defined `MAGENTO_BP`, so we shouldn't have an issue there.

As for if this file should be present in MFTF, I wanted to include it as an example. If you don't think we need one, then I'll happily get rid of it

Reply:
Going to delete this file.